wiki/content/deploy/dgraph-zero.md

```diff
@@ -6,51 +6,65 @@ weight = 8
 
-Dgraph Zero controls the Dgraph cluster. It automatically moves data between
-different Dgraph Alpha instances based on the size of the data served by each Alpha instance.
-
-It is mandatory to run at least one `dgraph zero` node before running any `dgraph alpha`.
-Options present for `dgraph zero` can be seen by running `dgraph zero --help`.
-
-* Zero stores information about the cluster.
-* `--replicas` is the option that controls the replication factor. (i.e. number of replicas per data shard, including the original shard)
-* When a new Alpha joins the cluster, it is assigned a group based on the replication factor. If the replication factor is 1 then each Alpha node will serve different group. If replication factor is 2 and you launch 4 Alphas, then first two Alphas would serve group 1 and next two machines would serve group 2.
-* Zero also monitors the space occupied by predicates in each group and moves them around to rebalance the cluster.
-
-Like Alpha, Zero also exposes HTTP on 6080 (+ any `--port_offset`). You can query (**GET** request) it
-to see useful information, like the following:
-
-* `/state` Information about the nodes that are part of the cluster. Also contains information about
-size of predicates and groups they belong to.
-* `/assign?what=uids&num=100` This would allocate `num` uids and return a JSON map
-containing `startId` and `endId`, both inclusive. This id range can be safely assigned
-externally to new nodes during data ingestion.
-* `/assign?what=timestamps&num=100` This would request timestamps from Zero.
-This is useful to fast forward Zero state when starting from a postings
-directory, which already has commits higher than Zero's leased timestamp.
-* `/removeNode?id=3&group=2` If a replica goes down and can't be recovered, you
-can remove it and add a new node to the quorum. This endpoint can be used to
-remove a dead Zero or Dgraph Alpha node. To remove dead Zero nodes, pass
-`group=0` and the id of the Zero node.
+Dgraph Zero controls the Dgraph cluster, and stores information about it. It
+automatically moves data between different Dgraph Alpha instances based on the
+size of the data served by each Alpha instance.
+
+Before you can run `dgraph alpha`, you must run at least one `dgraph zero` node.
+You can see the options available for `dgraph zero` using the following command:
+
+```bash
+dgraph zero --help
+
+`--replicas` is the option that controls the replication factor, the number
+ of replicas per data shard, including the original shard. When a new Alpha
+joins the cluster, it is assigned to a group based on the replication factor.
+If the replication factor is set to `1`, then each Alpha node will serve a
+different group. If the replication factor is set to `2` and you launch four
+ Alpha nodes, then first two Alpha nodes would serve group 1 and next two
+machines would serve group 2. Zero monitors the space occupied by predicates in
+each group and moves them around to rebalance the cluster.
+
```

Comment:
We actually can't set the replication factor to 2. It throws this error.
```
ERROR: Number of replicas must be odd for consensus. Found: 2
```
If the replication factor is set to `3` and you launch six Alpha nodes, then first three Alpha nodes would serve group 1 and next three machines would serve group 2. Zero monitors the space occupied by predicates in each group and moves them around to rebalance the cluster. Note that the replication factor has to be odd for consensus.